dspace-api/src/main/java/org/dspace/content/EntityServiceImpl.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content;
+
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.content.service.EntityService;
+import org.dspace.content.service.EntityTypeService;
+import org.dspace.content.service.ItemService;
+import org.dspace.content.service.RelationshipService;
+import org.dspace.content.service.RelationshipTypeService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class EntityServiceImpl implements EntityService {
+
+    @Autowired(required = true)
+    protected EntityTypeService entityTypeService;
+
+    @Autowired(required = true)
+    protected RelationshipService relationshipService;
+
+    @Autowired(required = true)
+    protected RelationshipTypeService relationshipTypeService;
+
+    @Autowired(required = true)
+    protected ItemService itemService;
+
+    public Entity findByItemId(Context context, UUID itemId) throws SQLException {
```

Comment:
please add the Override annotation also to simplify the check about the presence of javadoc for all the public methodds

Reply:
The Override annotation has been included